pom.xml

```diff
@@ -250,125 +250,6 @@
         <artifactId>rocksdbjni</artifactId>
         <version>${rocksdb.version}</version>
       </dependency>
-      <dependency>
-        <groupId>org.apache.hadoop</groupId>
-        <artifactId>hadoop-common</artifactId>
-        <version>${hadoop.version}</version>
-        <exclusions>
-          <exclusion>
-            <groupId>org.mortbay.jetty</groupId>
```

Comment:
do we not need to excluse all of those anymore?

Reply:
Moved those to `iceberg-views`